src/applications/personalization/profile360/actions/updaters.js

```diff
@@ -18,22 +17,26 @@ export const VET360_TRANSACTION_REQUEST_CLEARED = 'VET360_TRANSACTION_REQUEST_CL
 export const VET360_TRANSACTION_UPDATED = 'VET360_TRANSACTION_UPDATED';
 export const VET360_TRANSACTION_CLEARED = 'VET360_TRANSACTION_CLEARED';
 
-// function recordProfileTransaction(fieldName) {
-//   const names = [
-//     'mobile-phone',
-//     'primary-telephone',
-//     'mailing-address',
-//   ];
-//
-//   if (names.includes(fieldName)) {
-//     recordEvent({
-//       event: 'profile-transaction',
-//       'profile-section': fieldName
-//     });
-//   }
-// }
-
-export function refreshTransaction(transaction) {
+function recordProfileTransaction(event, fieldName) {
+  const analyticsMap = {
+    homePhone: 'home-telephone',
+    mobilePhone: 'mobile-telephone',
+    workPhone: 'work-telephone',
+    mailingAddress: 'mailing-address',
+    residentialAddress: 'home-address',
+    faxNumber: 'fax-telephone',
+    email: 'email',
+  };
+
```

Comment:
At some point we'll probably want to define a whole mapping in a config file or something.